ow_plexil/src/plexil-adapter/OwInterface.cpp

```diff
@@ -128,57 +128,45 @@ static vector<string> LanderOpNames = {
 
 ///////////////////////// Action Goal Status Support /////////////////////////
 
-// Repeating GoalStatus defined in actionlib_msgs/GoalStatus.msg
-// with the addition of a NOGOAL status for when the action is not running.
-static map<int, string> GoalStatus {
-  // GoalID -> Goal Status
-
-  { -1, "NOGOAL" },
-  { 0, "PENDING" },
-  { 1, "ACTIVE" },
-  { 2, "PREEMPTED" },
-  { 3, "SUCCEEDED" },
-  { 4, "ABORTED" },
-  { 5, "REJECTED" },
-  { 6, "PREEMPTING" },
-  { 7, "RECALLING" },
-  { 8, "RECALLED" },
-  { 9, "LOST" }
+// Duplication of actionlib_msgs/GoalStatus.h with the addition of a
+// NOGOAL status for when the action is not running.
+//
+enum ActionGoalStatus {
+  NOGOAL = -1u,
```

Comment:
'u' makes this an unsigned int, so this cannot actually represent **-1** in memory. Based on this blog post, https://brnz.org/hbr/?p=1433, it seems `-1u` converts a **-1** to an unsigned int, which yields the value of the maximum possible unsigned int. It is an interesting way to compute the maximum of an unsigned integer type, and this would not produce any errors, but I think it would be easy for a programmer to look at this and think that `NOGOAL == -1`, when in fact it does not.

Reply:
Sorry, that was a dumb error on my part.  I have simply removed the 'u' from all values.